### DISCUSSION TITLE
Write a short and descriptive title!

### DISCUSSION THREAD

**Discussion by MuCool1729** [typst/typst#4605] [General]
### Description

There seems to be a problem while compiling for a multi lettered subscript in math mode.

neither "$x_{n1}$" nor "$x_n1$" works, says n1 is unknown variable.

Tried the web app as well as tried compiling locally on WSL as well as using vscode extension, all seem problematic.

### Reproduction URL

https://typst.app/project/p0hwRIP62J7-6d3noC3CcB

### Operating system

Web app

### Typst version

- [x] I am using the latest version of Typst

**Comment by istudyatuni** [comment]
```typ
$x_(n 1)$
```
